iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,145 @@
+
+### Encryption
+
+If you need to use any secret configuration keys in the Bitrise build, encrypt the xcconfig files with those keys. Check that the encrypted xcconfig file is the one used by the test target.
+
+```
+# Put all the files in a tar file
+tar cvf secrets.tar file1 file2 
+
+# Encrypt the tar file
+openssl aes-256-cbc -k "password" -in secrets.tar -out secrets.tar.enc 
+```
+
+### Decryption
+
+Add add this as first step of the `bitrise.yml` (don’t forget to add ENCRYPT_PASSWORD to the bitrise secure keys):
+
+```
+    - script@1.1.3:
+        title: Decrypting files...
+        inputs:
+        - content: openssl aes-256-cbc -k "$ENCRYPT_PASSWORD" -in secrets.tar.enc -out secrets.tar -d && tar xvf secrets.tar
+```
+
+## Create Bitrise app...
+
+### Login in [Bitrise](https://www.bitrise.io/) using your own account. 
+
+If you’re not added to the `Wolox` organization, you should ask the technical leader to do so.
+
+### Add application
+
+In Bitrise dashboard, press "Add new app" to start the process. 
+
+### Choose project’s repository
+
+Before starting, make sure `Wolox` is selected in the "Add new app to" option in top left.
+
+![choose organization](./resources/br1.png)
+
+From the list of platforms, choose `Github`. Below, in the left part of the screen you can see a list of github organizations you contribute to. Choose `Wolox` (or the organization the repository is under), and then at the right choose the repository you want to integrate.
+
+![choose project](./resources/br2.png)
+
+### Setup repository access
+
+Select `auto-add SSH Key`. The key should be added automatically after a moment. If it doesn’t work, you may need to do it manually.
+
+NOTE: Only Github account admins can add automatically add the SSH key in this step. In case you have no access, skip this step and ask an admin to do it.
+
+![choose project](./resources/br3.png)
+
+### Validation setup
+
+Choose the main branch of your project (master or development usually). Bitrise will scan the branch to check for the correct configurations. This takes about 5 minutes.
+
+### Project build configuration
+
+Select iOS. Verify that the correct XCode stack is selected and confirm.
+
+![choose project](./resources/br4.png)
+
+### Webhook setup
+
+Select Register a new webhook for me. It will allow Bitrise to know when a `push` / `pull request` has been made, so it can test it. This step can only be done by an admin.
+
+NOTE: Only Github account admins can add automatically the webhook in this step. In case you have no access, skip this step and ask an admin to do it. It can be done from the Code tab in the project’s dashboard. Under “webhooks”, there’s a webhook URL which you can add to your Github repository following [this guide](https://github.com/bitrise-io/bitrise-webhooks#github---setup--usage).
+
+### Finish configuration
+
+Select We’ve kicked off your first test build for you to continue. 
```

Comment:
Maybe you could add what Capi wrote about, "it will probably fail, but don't worry because the bitrise configuration i not the correct one yet." or something like that.